airflow/providers/slack/operators/slack_webhook.py

```diff
@@ -169,6 +169,6 @@ def execute(self, context: Context) -> None:
             username=self.username,
             icon_emoji=self.icon_emoji,
             icon_url=self.icon_url,
-            # Unused Parameters, if not None than warn user
-            link_names=self.link_names,
+            # Unused Parameters, warn user if not None
+            **({"link_names": self.link_names} if self.link_names else {}),
```

Comment:
```suggestion
```

AFAIK I mark this value as not used for about I year, so we could just remove provide it in to the hook.